tasks/main.yml

```diff
@@ -1,27 +1,21 @@
 ---
 - name: Configure the Jenkins APT key
-  apt_key: url=http://pkg.jenkins-ci.org/debian/jenkins-ci.org.key
+  apt_key: url=http://pkg.jenkins.io/debian/jenkins.io.key
            state=present
 
 - name: Configure the Jenkins APT repositories
-  apt_repository: repo="deb http://pkg.jenkins-ci.org/debian binary/"
-                  state=present
+  apt_repository:
+    repo: deb http://pkg.jenkins-ci.org/debian binary/
+    state: present
+    update_cache: yes
 
 - name: Install Jenkins
```

Comment:
This task appears to be failing consistently on 14.04. I think it may be because Java isn't fully installed by the time Jenkins tries to do its thing. At least, if I install Java in a separate task, Jenkins installs cleanly.

Have you tried adding the `azavea.java` role as a dependency to this role so that Jenkins doesn't try to install Java itself?